packages/node_modules/@webex/widget-space/src/container.js

```diff
@@ -143,19 +145,49 @@ export class SpaceWidget extends Component {
       const callStartTime = (call?.activeParticipantsCount > 0) ? call?.startTime : null;
 
       let menuButton;
+      const {preferredWebexSite} = sparkInstance.meetings;
+      let isMeetButtonDisabled = false;
+
+      if (sparkInstance.meetings.getMeetingByType(CONSTANTS.CONVERSATION_URL, currentConversation)) {
+        isMeetButtonDisabled = false;
+      }
+      else {
+        isMeetButtonDisabled = !preferredWebexSite;
+      }
+      let errorElementSpace = null;
+
+      if (isMeetButtonDisabled) {
+        const widgetError = {
+          temporary: false,
+          title: formatMessage(messages.noAbilityToStartMeeting)
+        };
+
+        errorElementSpace = (
+          <div className={classNames('webex-error-wrapper', styles.errorWrapper)}>
+            <ErrorDisplay
+              transparent={widgetError.temporary}
+              {...widgetError}
```

Comment:
We can directly use the properties of `widgetError`. Creating an object and then spreading here doesn't seem to be required as it's being used only once.

Reply:
@rarajes2, we can also use it directly. I just kept separate `widgetError` because they are following in the code.

Reply:
@rarajes2, Done.